dask_expr/tests/test_groupby.py

```diff
@@ -84,7 +84,8 @@ def test_groupby_nunique(df, pdf):
     with pytest.raises(AssertionError):
         df.groupby("x").nunique()
 
-    assert_eq(df.groupby("x").y.nunique(), pdf.groupby("x").y.nunique())
+    assert_eq(df.groupby("x").y.nunique(split_out=1), pdf.groupby("x").y.nunique())
+    assert_eq(df.groupby("x").y.nunique(split_out=True), pdf.groupby("x").y.nunique())
```

Comment:
Not a blocker, but I think it might be nice to expand these checks slightly. For example, we may want to validate the actual partition count. E.g.

```python
assert df.groupby("x").y.nunique().npartitions == df.npartitions
```

Also, can we check that `split_out: Callable` is working?